colorize.js

```diff
@@ -271,6 +275,14 @@ function hasId (element, id) {
   return element.innerHTML.includes(id)
 }
 
+function isErrorLineGeneral (element) {
+  return element.innerHTML.toLowerCase().includes(errorLevel)
+}
+
+function isDebugOrInfo (element) {
+  return element.innerHTML.toLowerCase().includes(infoLevel) ||  element.innerHTML.toLowerCase().includes(debugLevel)
```

Comment:
```suggestion
  const text = element.innerHTML.toLowerCase()
  return text.includes(infoLevel) ||  text.includes(debugLevel)
```
You can also refactor code if there are similar ones.

Reply:
done